.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,44 +1,53 @@
 ---
 name: Bug report
 about: Report bugs and errors found while using the Terraform Provider.
-title: ''
+title: ""
 labels: bug
-assignees: ''
-
+assignees: ""
 ---
 
 ### Your environment
 
 <!-- Version of the Terraform Provider when the error occurred -->
+
 Terraform Provider Version:
 
-<!-- What version of the Connect server are you running?
+<!-- What authentication method are you using?
+- Connect: Using Connect server credentials (url + token)
+- Service Account: Using service account token
+- Biometric/Desktop: Using account name/ID for biometric unlock
```

Comment:
To keep this consistent with how we call this in [dev portal docs](https://developer.1password.com/docs/sdks/desktop-app-integrations)
```suggestion
- Desktop app: Using account name/ID for biometric unlock
```